doc/md/fundamentals/2-types/10-options-results.md

```diff
@@ -180,13 +180,10 @@ In the case of a `Result`, you can also use pattern matching with the difference
 
 ```motoko no-repl
 func greetResult(resultName : Result<Text, Text>) : Text {
-  switch (resultName) {
-    case (#err(error)) { "No name: " # error };
-    case (#ok(name)) { "Hello, " # name };
-  }
+  let #ok(name) = resultname else return #err("No Name")
 };
 assert(greetResult(#ok("Motoko")) == "Hello, Motoko!");
-assert(greetResult(#err("404 Not Found")) == "No name: 404 Not Found");
+assert(greetResult(#err("404 Not Found")) == "No name");
 ```
```

Comment:
This example is bit weird since it takes a `Result` as an argument. 
Maybe just do safeDivide with a Result?
``` motoko
func safeDivide(a : Int, b : Int) : Result<Int,Text> {
    if (b == 0) (#err "Division by 0"  else (#ok (a / b));
}

let result1 = safeDivide(10, 2); // #ok 5
let result2 = safeDivide(10, 0); // #err "Division by 0"
```